javascript/javascript3/week2/Tasks/script.js

```diff
@@ -0,0 +1,41 @@
+
+//Promise that resolves after set time
+function resolveAfterSeconds(resolveAfter){
+    return new Promise((resolve)=>{
+        setTimeout(()=>{
+            resolve();
+        },resolveAfter*1000)
+    })
+}
+
+resolveAfterSeconds(3).then(()=>{
+    console.log("Resolved");
+});
+
+//Fetching and waiting
+const APIKEY = "9469946e53df5e8174588db17f99cd64";
+
+const promiseFetch = new Promise((resolve, reject)=>{
```

Comment:
Small detail but the task is to create a function that returns a Promise, not directly creating a Promise.

The difference is that the Promise starts executing as soon as it is defined, while if it's a function that returns a Promise the Promise doesn't start executing until the function is called.